data/module_source/code_execution/Invoke-Assembly.ps1

```diff
@@ -0,0 +1,51 @@
+# Modified version of Invoke-Assembly
```

Comment:
I would add a small thing at the beginning to give some context to the script. You can use [this](https://github.com/BC-SECURITY/Invoke-PrintDemon/blob/master/Invoke-PrintDemon.ps1) as an example. Also, another opportunity for you to claim credit as the author.